ZenPacks/zenoss/ZenPackLib/tests/ZPLTestBase.py

```diff
@@ -132,13 +132,11 @@ class ZPLTestMockZenPack(BaseTestCase):
     yaml_doc = '''name: ZenPacks.zenoss.ZenPackLib'''
     zenpack_name = 'ZenPacks.zenoss.MockZenPack'
     disableLogging = True
-    datsources = []
-    thresholds = []
 
-    def afterSetUp(self):
+    def afterSetUp(self, datasources=[], thresholds=[]):
```

Comment:
Mutable default arguments are just as bad a mutable class-level property values. You should do this instead.

```
def afterSetUp(self, datasources=None, thresholds=None):
    if datasources is None:
        datasources = []

    if thresholds is None:
        thresholds = []
```

Reply:
Would you also fix `MockZenPack.__init__` in the same way?